### PR TITLE
Create strings.xml

### PR DESCRIPTION
**Pull request by Fish25op**
Azerbaijani translations. It will be 100% completed later🥰

### CODE REVIEW

File: app/src/main/res/values-az/strings.xml [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+    <string name="account_settings_show_avatars">Avatarları göstər</string>
+    <string name="account_settings_show_bot_accounts">Bot hesabların göstər</string>
+    <string name="account_settings_show_notifications_for_new_posts">Yeni elanlar üçün bildirişlər göstər</string>
+    <string name="account_settings_show_nsfw">NSFW Göstər</string>
+    <string name="account_settings_show_read_posts">Oxunan elanları göstər</string>
+    <string name="account_settings_show_scores">Xalları göstər</string>
+    <string name="account_settings_subscribed">Abunə olundu</string>
+    <string name="addBookmark">Əlfəcinlərə əlavə et</string>
+    <string name="app_bars_dot_spacer">·</string>
+    <string name="app_name" translatable="false">Jerboa</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:34]: Remove this one

File: app/src/main/res/values-az/strings.xml [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    <string name="account_settings_send_notifications_to_email">E-poçta bildirişlər göndər</string>
+    <string name="account_settings_show_avatars">Avatarları göstər</string>
+    <string name="account_settings_show_bot_accounts">Bot hesabların göstər</string>
+    <string name="account_settings_show_notifications_for_new_posts">Yeni elanlar üçün bildirişlər göstər</string>
+    <string name="account_settings_show_nsfw">NSFW Göstər</string>
+    <string name="account_settings_show_read_posts">Oxunan elanları göstər</string>
+    <string name="account_settings_show_scores">Xalları göstər</string>
+    <string name="account_settings_subscribed">Abunə olundu</string>
+    <string name="addBookmark">Əlfəcinlərə əlavə et</string>
+    <string name="app_bars_dot_spacer">·</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:33]: Make the original not translatable, and remove all occurrences of this in the other translations.

**Review by MV-GH — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-az/strings.xml [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+    <string name="community_subscribe">Abunə ol</string>
+    <string name="community_users_month">%1$s istifadəçi / ay</string>
+    <string name="crash_logs">Çökmə Jurnalları</string>
+    <string name="crash_logs_all_deleted">Bütün çökmə jurnalları silindi</string>
+    <string name="crash_logs_copy">Çökmə jurnalın köçür</string>
+    <string name="crash_logs_delete">Bütün çökmə jurnalların sil</string>
+    <string name="createPost_selectCommunity">Siyahıdan icma seç</string>
+    <string name="create_post_a_title_here">burada bir başlıq ...</string>
+    <string name="create_post_community">İcma</string>
+    <string name="create_post_copy_suggested_title"> %1$s<> təklif olunan başlığı köçür</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:112]: Remove the <>

File: app/src/main/res/values-az/strings.xml (hunk 1 above)
Comment (Fish25op) [reply app/src/main/res/values-az/strings.xml:34]: Why?
Whole?

File: app/src/main/res/values-az/strings.xml [hunk 4]
```diff
@@ -31,7 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="app_bars_dot_spacer">·</string>
-    <string name="app_name" translatable="false">Jerboa</string>
+    <string name="app_name">Jerboa</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:34]: Remove the entry not the attribute

File: app/src/main/res/values-az/strings.xml [hunk 5]
```diff
@@ -31,7 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="app_bars_dot_spacer">·</string>
-    <string name="app_name">Jerboa</string>
+    <string name="app_name" translatable>Jerboa</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:34]: No, I mean the whole string. When a entry isn't found in a translation it falls back to the default. So it isn't needed here. Same with all the English entries.

File: app/src/main/res/values-az/strings.xml [hunk 6]
```diff
@@ -30,7 +30,6 @@
     <string name="account_settings_show_scores">Xalları göstər</string>
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
-    <string name="app_bars_dot_spacer">·</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml]: Mark the original default non translatable

File: app/src/main/res/values-az/strings.xml [hunk 7]
```diff
@@ -31,6 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="bottomBar_bookmarks">Əlfəcinlərimə keç</string>
+    <string name="app_bars_dot_spacer">·</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:34]: No, i mean remove it here but modify the strings.xml in values to add the translatable="false"

File: app/src/main/res/values-az/strings.xml [hunk 8]
```diff
@@ -31,7 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="bottomBar_bookmarks">Əlfəcinlərimə keç</string>
-    <string name="app_bars_dot_spacer">·</string>
+    <string name="app_bars_dot_spacer" translatable="false">·</string>
```
Comment (MV-GH) [inline app/src/main/res/values-az/strings.xml:34]: Almost there, now remove this line here and add it in values/strings.xml (replace the old line there)

**Comment by MV-GH** [comment]
Remove `app_bars_dot_spacer` from all the other translations. Such as `values-da\strings.xml`. Then ill approve it.

**Comment by Fish25op** [comment]
I didn't understand

**Review by dessalines — APPROVED** [review]

**Pull request merged**